browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -35,6 +37,13 @@ BraveNewTabUI::BraveNewTabUI(content::WebUI* web_ui, const std::string& name)
           web_ui,
           true /* Needed for legacy non-mojom message handler */),
       page_factory_receiver_(this) {
+  const bool was_restored =
+      web_ui->GetWebContents()
+          ->GetController()
+          .GetLastCommittedEntry()
+          ->IsRestored() &&
+      web_ui->GetWebContents()->GetVisibility() != content::Visibility::VISIBLE;
```

Comment:
@iambrianfung when restoring new tab pages on browser launch, if the active tab is a blank new tab page should we rotate the index? If not I will remove this change, and if so I will refactor this change